specs/receive_funds/01_main.rst

```diff
@@ -0,0 +1,89 @@
++-------+---------------+-----------+------------+
+| issue | title         | author    | created    |
++-------+---------------+-----------+------------+
+| 68_   | Receive funds | tschubotz | 2019-01-17 |
++-------+---------------+-----------+------------+
+
+.. _68: https://github.com/gnosis/safe/issues/68
+
+Receive funds
+===============
+
+1. `Main`_
+2. `User Interface`_
+3. `External Communication`_
+4. Other_
+
+.. _Main:
+
+.. contents:: Table of Contents
+    :depth: 3
+
+1. Problem Definition
+---------------------
+
+- User wants to receive funds
+- User should be able to share their wallet address via different ways
+    - Via QR code for scanning.
+    - Copy to clipboard.
+    - Via other services on their phone.
+- There should be a way to double check if the other person has the
+  correct shared address.
+
+2. Inputs
+-----------
+
+- Address of the currently selected Safe.
+
+3. Outputs
+------------
+
+- Address is clearly visible.
+- Identicon is visible.
+- QR code can be scanned.
+- Address can be shared via other apps
+- Address can by copied to clipboard.
+
+
+4. Use Case Scenarios
+-----------------------
+
+4.1. Happy Case
+~~~~~~~~~~~~~~~
+
+Preconditions
++++++++++++++
+
+- Safe exists on the device.
+
+Steps
++++++
+
+1. Open receive screen
+2. See address
+3. See identicon
+4. Copy address to clipboard
```

Comment:
I left it general here so the UI design can decide more freely how to implement it and we don't have to always update the use cases.

Reply:
But this is something I would require before merging, since this is not really visible in the UI.

In the case of the QR code I get your reasoning, but here if we use a single-click or long-press is something we don't see in UI layouts.

I would add an explicit "Interaction method TBD" or something like it here

Reply:
Well I thought in the UI we have a label that says "Long tap to copy" :)
If we wouldn't have that, I agree.

Reply:
ok, fair